### PR TITLE
Fix editorial issues in Core Properties section.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/630.html" title="Last updated on Feb 16, 2021, 2:46 PM UTC (8a4dbae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/630/9b373a1...8a4dbae.html" title="Last updated on Feb 16, 2021, 2:46 PM UTC (8a4dbae)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1303,274 +1303,230 @@ <h2>Extensibility</h2>
...
-  The property names <code>id</code>, <code>type</code>, and
-  <code>controller</code> can be present in different classes with possible
-  differences in constraints.
-      </p>
+    </p>
+    <p class="note" title="Property names used in multiple classes">
+The property names <code>id</code>, <code>type</code>, and
+<code>controller</code> can be present in different classes with possible
+differences in constraints.
+    </p>
```
Comment (rhiaro) [inline index.html:1325]:
```suggestion
    </p>
    <p class="note" title="Property names used in maps of different types">
The property names <code>id</code>, <code>type</code>, and
<code>controller</code> can be present in maps of different types
with possible differences in constraints.
    </p>
```

With the "three classes of core properties" text removed, "classes" here no longer reads well.

**Review by rhiaro — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]
I agree with the simplification here and the removal of the "three classes of core properties" language (and with @rhiaro 's comment on that).

If possible we should wait for @iherman 's review since I think he wrote most of this section.

**Review by iherman — CHANGES_REQUESTED** [review]
I have no problems with the simplification of the section; it was mostly an introductory paragraph for the tables with the properties.

However. The change also removed the note whereby in the DID model, the order of the items in an ordered set are irrelevant. What this means is that, if we follow INFRA, the order of, say, verification methods has a meaning (which is not the case). If order is meaningful than the vocabularies and the constraints are also to be re-done, using RDF lists instead of just repeating property values. 

I just checked and the note was the only place in the document where this fact was mentioned. This note should be reinstated in the text.

**Comment by msporny** [comment]
> However. The change also removed the note whereby in the DID model, the order of the items in an ordered set are irrelevant.

The statement was moved to a separate section in the specification in PR #631 -- see https://github.com/w3c/did-core/pull/631/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R1276

**Comment by iherman** [comment]
> The statement was moved to a separate section in the specification in PR #631 -- see https://github.com/w3c/did-core/pull/631/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R1276
> 
> PS: I'm really glad that reviewers are paying attention to and are catching these potential oversights. :)

Sigh... Such tricks make our life difficult 😄

I am fine merging this PR *and* #631 together... 😉

**Comment by msporny** [comment]
@iherman wrote:
> Such tricks make our life difficult smile

Just making sure everyone is paying attention... :P

**Review by iherman — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**